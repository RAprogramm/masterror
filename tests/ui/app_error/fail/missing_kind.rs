// SPDX-FileCopyrightText: 2025 RAprogramm <andrey.rozanov.vl@gmail.com>
//
// SPDX-License-Identifier: MIT

use masterror::Error;

#[derive(Debug, Error)]
#[error("oops")]
#[app_error(message)]
struct MissingSpec;

fn main() {}
