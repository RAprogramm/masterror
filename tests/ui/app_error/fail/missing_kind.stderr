error: missing `kind = ...` in #[app_error(...)]
 --> tests/ui/app_error/fail/missing_kind.rs:9:1
  |
9 | #[app_error(message)]
  | ^
